packages/frontend-utils/package.json

```diff
@@ -14,7 +14,8 @@
   },
   "peerDependencies": {
     "react": "^17.0.2",
-    "react-router-dom": "^5.2.0"
+    "react-router-dom": "^5.2.0",
```

Comment:
is this a dependency we don't want to pin?

Reply:
to pin? what do you mean?

Reply:
We have this change recently where it removes `^` from versions. https://github.com/yldio/asap-hub/commit/c4e964bf771f6423f4c692088b864c9764dc8927
I wonder if it should be the same for react-router-dom